data/humans/occupations.json

```
@@ -235,6 +235,7 @@
 		"director",
 		"dishwasher",
 		"dispatcher",
+    "dj",
```

```suggestion
    "DJ",
```
Let's put this in uppercase. Thanks!